app/assets/javascripts/discourse/app/components/plugin-outlet.hbs

```diff
@@ -4,7 +4,7 @@
     But for now, this classic component wrapper takes care of the tagName.
   }}
   <this.wrapperComponent @tagName={{@tagName}}>
-    {{#each this.connectors as |c|}}
+    {{#each (this.getConnectors) as |c|}}
```

Comment:
why do we need () here?

Reply:
The brackets turn the simple property reference into a 'helper invocation'. And since [plain old functions can now be helpes](https://blog.emberjs.com/plain-old-functions-as-helpers/) yes, exactly as you said, it makes it a function invocation (with autotracking support!) 🚀

Reply:
Dump question: what are "connectors" in this context?